### PR TITLE
[Site Initializer] LPS-159613 update StructuredContentFolder

### PR DESCRIPTION
**Pull request by nNilton**

### CODE REVIEW

**Comment by nNilton** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-extender/site-initializer-extender-test-bundle/src/main/resources/site-initializer/journal-articles/test-journal-article-1.metadata.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
-	"description": "This is the description for Test Journal Article 1.",
-	"name": "Test Journal Article 1",
+	"description": "This is the description for Test Journal Folder 1.",
+	"externalReferenceCode": "JOURNALFOLDER001",
+	"name": "Test Journal Folder 1",
 	"viewableBy": "Anyone"
 }
```
Comment (gleicelisbino) [inline modules/apps/site-initializer/site-initializer-extender/site-initializer-extender-test-bundle/src/main/resources/site-initializer/journal-articles/test-journal-article-1.metadata.json:6]: Change the file names.

**Comment by nNilton** [comment]
ci:test:sf

**Comment by gleicelisbino** [comment]
ci:forward

**Comment by nNilton** [comment]
ci:test:relevant

**Pull request closed without merging**